package.json

```diff
@@ -33,6 +33,7 @@
     "linefeed"
   ],
   "devDependencies": {
-    "mocha": "*"
+    "ava": "^0.14.0",
+    "xo": "^0.15.1"
```

Comment:
Use star `*` versions on these. I do that on my own projects to be able to catch possible bugs in those early. Better me having to fix breaking stuff than my users.